lib/perl/Genome/InstrumentData/AlignmentResult/Merged.pm

```diff
@@ -707,4 +786,50 @@ sub scalar_property_from_underlying_alignment_results {
 }
 
 
+sub _lock_per_lane_alignment {
+    my $self = shift;
+
+    for my $alignment ($self->collect_individual_alignments) {
+        my $id = $alignment->id;
+        unless ($alignment->get_merged_alignment_results) {
+            my @bams = $alignment->alignment_bam_file_paths;
+            unless (@bams) {
+               die $self->error_message("Per lane alignment $id has neither merged results nor bam paths. Please create an apipe-support ticket for this.");
```

Comment:
Maybe give some more detail as to what needs to be done, i.e., that the instrument data needs to be realigned.